formal/symbiyosys/build.sh

```diff
@@ -0,0 +1,105 @@
+#!/bin/bash
+#****************************************************************************
+#* build.sh
+#* 
+#* Conda package build script for SymbiYosys and supporting solvers
+#****************************************************************************
+
+set -e
+set -x
+
+if [ x"$TRAVIS" = xtrue ]; then
+	CPU_COUNT=2
+fi
+
+cwd=`pwd`
+
+CFLAGS_ORIG=${CFLAGS}
+
+#********************************************************************
+#* Build Boolector
+#********************************************************************
+cd ${cwd}
+
+# CFLAGS=`echo ${CFLAGS_ORIG} | sed -e 's/-O[0-9]//g'`
+unset CFLAGS
+
+git clone https://github.com/Boolector/boolector.git
+
+cd ${cwd}/boolector
+
+# ilingeling.c contains a variable named 'clone'
+# This conflicts with the clone function, so we rename it
+sed -i -e 's%cd ${LINGELING_DIR}%cd ${LINGELING_DIR}\nsed -i -e "s/\\<clone\\>/_clone/g" ilingeling.c%' contrib/setup-lingeling.sh
+
+echo "CFLAGS=${CFLAGS}"
+./contrib/setup-btor2tools.sh
+if test $? -ne 0; then exit 1; fi
+
+./contrib/setup-lingeling.sh
+if test $? -ne 0; then exit 1; fi
+
+echo "CFLAGS=${CFLAGS}"
+./configure.sh --prefix ${PREFIX}
+if test $? -ne 0; then exit 1; fi
+
+cd build
+
+make -j${CPU_COUNT}
+if test $? -ne 0; then exit 1; fi
+
+make install
+if test $? -ne 0; then exit 1; fi
+
+CFLAGS=${CFLAGS_ORIG}
+
+#********************************************************************
+#* Build Z3
+#********************************************************************
+cd ${cwd}
+
+git clone https://github.com/Z3Prover/z3.git z3
```

Comment:
This should be part of the meta.yaml?

Reply:
See https://docs.conda.io/projects/conda-build/en/latest/resources/define-metadata.html#source-from-multiple-sources

Reply:
Nice! Somehow, even after searching I didn't manage to find that description of how to use multiple sources.

Reply:
Moved this to meta.yaml